lib/game.js

```diff
@@ -14,16 +14,17 @@ Game = {
 		}
 	},
 
-	create_player: function(socket){
+	create_player: function(socket, player_id){
```

Comment:
How do you have a player_id before creating a player? I think this function is mis-named.